yahooquery/base.py

```diff
@@ -947,24 +948,13 @@ def symbols(self):
     def symbols(self, symbols):
         self._symbols = _convert_to_list(symbols)
         
-    def get_crumb(self):
+    def set_crumb(self):
         response = self.session.get('https://query2.finance.yahoo.com/v1/test/getcrumb')
```

Comment:
Hi Shall we enable proxy here as well?

Reply:
The proxies are already set on the session when it's being [initialized](https://github.com/dpguthrie/yahooquery/blob/cb5b2bd5a7e351ba29414aca3259815e8b4ead6c/yahooquery/utils/__init__.py#L57).